### ISSUE TITLE
Unlearning Engineering specialization

### ISSUE THREAD

**Issue by Phatcat** [cmangos/classic-db#7]
https://www.youtube.com/watch?v=-KYeae_HMv4

This option is missing from cmangos. May need some core suppot we currently don't provide.

**Comment by Phatcat** [comment]
https://github.com/cmangos/wotlk-db/issues/26

**Comment by cala** [comment]
Hmm. I don't think that Engineering specialisation could be unlearned in Classic. 
At least, I'm sure it was not possible to unlearn Blacksmithing or Leatherworking specialisation in Classic and I don't see why Engineering would be different.

**Comment by Phatcat** [comment]
Yes, you could unlearn engineering specialization in classic on retail back in 2006 and even 2005. Also worked for leatherworking.
https://blue.mmo-champion.com/topic/16842-eng-tinkers-guide-to-engineering/ (from 2006)
>As of 1.10, you can delete engineering and take it up again to take up Goblin/Gnome engineering. Before 1.10, you could only take up the side you'd chosen at first. However, when rechoosing a side, you'll need to relevel engineering completely. If you have taken up a specialization, and dropped engineering again, visit Narain Soothfancy in his house north of Steamwheedle Port, Tanaris. Click the book "Soothsaying for Dummies" on his table, and voilá!

Wowpedia from 2006: https://wow.gamepedia.com/index.php?title=Soothsaying_for_Dummies&oldid=344872
>Reading this book removes previous commitment to a discipline specialization in a profession that has been completely abandoned. This enables professionals to choose a different discipline specialization in a profession they had previously known. 

wow patch notes 1.10: https://wowwiki.fandom.com/wiki/Patch_1.10.0
>Professions
> Players that unlearn Engineering or Leatherworking may now respecialize if they decide to relearn the professions.

allakhazam comments from 2005 (first one doesn't matter and is from 2005 before the item did anything, the others are from april and august 2006); https://web.archive.org/web/20071105061642/http://wow.allakhazam.com/db/mob.html?wmob=11811
> There's a book next to him "Soothsaying for Dummies" but nothing happens after you click on it. 
>
> What quest is involved with him? 
>>The book if for changing leather working profession once you already chose one. (Dragonscale, tribal and elemental)
>>>Also works for engineering.
>>>
>>>
>>>Its such a pain in the ass to find.

**Comment by Phatcat** [comment]
Do not ask me why blacksmiths didn't have the same opportunity, though, that one is beyond me.

**Comment by Phatcat** [comment]
In 2006 you had to abandon the profession first though unlike in the video where it seems you can simply drop the specialization and pick up a new one all whilst retaining your profession and profession skill level, back in classic you would have had to drop your profession and with that lose all your recipies, level your professoin back up to... is it 200 or 225, I don't recall, and then you could re-do the specialization quest as if you'd never done it I'd imagine. Things are a little sparse on info but it was possible indeed for engineers and apparently also leatherworkers.

This is guesswork, but I am thinking the gold price is a price that was added later instead when it was changed to you being able to retain the skill while losing the specialization.

**Comment by cala** [comment]
Thank you for this research!
